app/pages/ConfigurationPage/ConfigurationPage.tsx

```diff
@@ -152,94 +188,105 @@ const SavePasswordStep = (props: { prevStep: () => void; onPasswordSave: (hasAgr
+              control={<Radio />}
+              label="Pamiętaj"
+            />
+            <FormControlLabel
+              onChange={() => setHasAgreed(false)}
+              checked={!hasAgreed}
+              control={<Radio />}
+              label="Nie pamiętaj"
+            />
+            <Space size={1.5} />
+            <Button
+              onClick={() => onPasswordSave(hasAgreed)}
+              glow
+              color={THEME.colors.palette.teal.main}
+              variant="primary"
+              fullWidth
+              disabled={!hasAgreed}
+            >
+              Gotowe
+            </Button>
+          </CenterContent>
+          <FooterInfo color={THEME.colors.palette.teal.light}>
+            Jak w przeglądarce zapisujemy dane logowania, żeby nie wpisywać ich za każdym razem. Kto chciałby wpisywać
+            to samo co 5 minut? Oczywiście wszystko jest bezpieczne, dane logowania zapisywane są w tzw. bezpiecznej
+            enklawie twojego systemu operacyjnego, miejsca specjalnie przeznaczonego do zapisywania haseł.
+          </FooterInfo>
+        </SidebarContent>
       </StyledSidebar>
       <ConfigurationMockup color={THEME.colors.palette.teal.light} />
     </Box>
   )
 }
 
 const CongratulationsStep = ({ onConfigurationExit }: { onConfigurationExit: () => void }) => (
-  <Box width="100vw" height="100vh" overflow="hidden" position="relative" display="flex">
-    <StyledSidebar style={{ minWidth: 425, maxWidth: 500, width: '50%' }}>
-      <Text fontWeight="bold" size={2}>
-        Wszystko gotowe!
-        <span role="img" aria-label="gwiazdki">
-          ✨
-        </span>
-      </Text>
-      <Text size="16px" color="#2B2B2B">
-        Garśc informacji od twórców:
-      </Text>
-      <Space size={5.5} />
-      <Text fontWeight={700} size="20px">
-        <span role="img" aria-label="kawa">
-          ☕️
-        </span>
-        Aplikacja jest w trakcie rozwoju
-      </Text>
-      <Text>
-        ...więc mogą pojawić się błędy. Jesteśmy tylko studentami, a aplikację tworzymy po godzinach nauki i pracy. Poza
-        tym - walka z systemami politechniki to nie lada wyzwanie{' '}
-        <span role="img" aria-label="uśmiechnięta buźka">
-          😅
-        </span>
-      </Text>
-      <Space size={2} />
-      <Text fontWeight={700} size="20px">
-        <span role="img" aria-label="ziemia">
-          ‍🌍
-        </span>
-        Kod dostępny jest publicznie
-      </Text>
-      <Text>
-        Programistów zachęcamy do odwiedzenia{' '}
-        <Link url="https://github.com/knnativeflow/zdalnie-na-pwr">naszego GitHuba</Link> i dodania czegoś od siebie.
-      </Text>
-      <Space size={2} />
-      <Space size={5} />
-      <Button
-        color={THEME.colors.palette.pink.main}
-        variant="primary"
-        glow
-        onClick={onConfigurationExit}
-        style={{ alignSelf: 'center' }}
+  <Box>
+    <StyledSidebar>
+      <SidebarContent
+        css={css`
```

Comment:
Ten props `css` działa? Według nich https://emotion.sh/docs/css-prop trzeba dorzucić jeszcze coś, żeby działał.

Reply:
Nawet nie wiedziałem 😆 Zadziałał, a jak sprawdziłem to jest dodany jakiś plugin z emotion do babela 😮

Reply:
Ooo a to znaczy, że @karolsitarz się postarał :D